cmd/crates/stellar-ledger/src/emulator_tests.rs

```diff
@@ -4,9 +4,7 @@ use soroban_env_host::xdr::Transaction;
 use soroban_env_host::xdr::{self, Operation, OperationBody, Uint256};
```

Comment:
Do we need this file here anymore? Could it too be moved to `tests/test`

Reply:
Good point! I'll move this to `tests`. I may leave it at the top level of `tests` since it is the only integration test file so far.

Reply:
Oh actually, i see now how that's being done in `soroban-test` - i like that! It also makes it a bit easier to turn on/off the emulator-tests via a feature.